demos/general/demo_behavior.py

```diff
@@ -42,9 +45,9 @@
 
 #%%
 def main():
-    pass # For compatibility between running under Spyder and the CLI
+    pass # For compatibility between running under IDE and the CLI
 
-#%%
+    #%%
     pl.ion()
 
     fname = [u'demo_behavior.h5']
```

Comment:
With py3, the u marker on the string is no longer necessary (I know you didn't touch this, it's just in view.

We also could remove all the spyder breakpoints if we want, converting any that are comments into regular comments, or we could just leave them. Not that important to adjust either, really.

Reply:
I considered this, but these code cells are now used in VS Code and even Pycharm, so I think they are useful. For people in CLI it doesn't really matter, but in the IDE they can be pretty helpful.

Reply:
Fair enough.